nnd-data-poll-service/src/main/java/gov/cdc/nnddatapollservice/repository/rdb_modern/model/NrtObservation.java

```diff
@@ -300,9 +318,14 @@ public NrtObservation(NrtObservationDto dto) throws ParseException {
         this.healthCareId = dto.getHealthCareId(); // Added this field
         this.morbHospReporterId = dto.getMorbHospReporterId(); // Added this field
         this.priorityCd = dto.getPriorityCd(); // Added this field
+        this.transcriptionistAuthType= dto.getTranscriptionistAuthType();
+        this.transcriptionistIdAssignAuth= dto.getTranscriptionistIdAssignAuth();
+        this.assistantInterpreterAuthType= dto.getAssistantInterpreterAuthType();
+        this.assistantInterpreterIdAssignAuth= dto.getAssistantInterpreterIdAssignAuth();
+        this.accessionNumber= dto.getAccessionNumber();
+        this.morbHospId= dto.getMorbHospId();
```

Comment:
Looks like NrtObservation  still does not take all the NrtObservationDto fields?
What about `reportObservationUid`, `followupObservationUid`, `reportRefrUid`, `reportSprtUid`, `morbPhysicianId`, `morbReporterId`.